documentation/hang/step2/using_inspector.md

```diff
@@ -1 +1,54 @@
-// TODO
+# Using Inspectors
+
+Inspectors are very useful on development. It allow us to step into our
+programs and get our hands on every slice of our code and values of its
+local variables.
+
```

Comment:
Add a paragraph mentioning that the Inspector is built to be production safe, as long as properly used (which means **not** connecting Chrome Devtools to a production instance). Later we can add a link on this paragraph to a new session "Using the Inspector Protocol in production"